schema.sql

```diff
@@ -1,17 +1,21 @@
 /* Database schema to keep the structure of entire database. */
 
--- Add the species column to the animals table
+-- Create the owners table
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    full_name VARCHAR(255),
+    age INTEGER
+);
+
+-- Create the species table
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255)
+);
+
+-- Modify the animals table
 ALTER TABLE animals
-ADD COLUMN species VARCHAR(255);
+ADD COLUMN species_id INTEGER REFERENCES species(id),
+ADD COLUMN owner_id INTEGER REFERENCES owners(id);
```

Comment:
- [ ] As stated in the project requirement Kindly make sure that Modify animals table by Remove column species So kindly write the Sql for Removing column species just like the below sample. 

   ```
ALTER TABLE animals 
  DROP column species;
  ADD COLUMN species_id INTEGER REFERENCES species(id),
 ADD COLUMN owner_id INTEGER REFERENCES owners(id);
```